tests/functional/WebDriverTestCase.php

```diff
@@ -39,7 +39,9 @@ protected function setUp(): void
             if (getenv('BROWSER_NAME')) {
                 $browserName = getenv('BROWSER_NAME');
             } else {
-                $browserName = WebDriverBrowserType::HTMLUNIT;
+                $this->markTestSkipped(
```

Comment:
here you could invert the condition and call markTestSkipped because it throws and would stop execution

Reply:
True, thanks